marconi-sidechain-node/marconi-sidechain-node/README.md

```diff
@@ -0,0 +1,2 @@
+# marconi-sidechain-node
+
```

Comment:
Maybe add a line as "under development, can't be used yet" or anything that allows people to know that they probably don't want to play with it. As we plan to do a proper release, we may want to be clear about it.